src/bot.ts

```diff
@@ -556,20 +556,26 @@ async function withRetries<T>(
     const INITIAL_DELAY = 100; // ms
     let delay = INITIAL_DELAY;
     while (!result.ok) {
-        let mustDelay = true;
+        let mustDelay = false;
```

Comment:
Guess there should be a comment for someone like me in the future… 😬

```suggestion
        let mustDelay = false; // handled in `finally`
```